Dockerfile

```diff
@@ -0,0 +1,44 @@
+FROM ubuntu:14.04
+
```

Comment:
add "ENV DEBIAN_FRONTEND noninteractive" to supresses loads of warnings, less clutter